locales/en/events.json

```diff
@@ -362,7 +362,7 @@
     "description": ""
   },
   "PersonMoveDeathInCustody": {
-    "flag": "red",
+    "flag": "grey",
```

Comment:
I'm actually wondering if these are still used, I noticed some of the events don't have one.

Reply:
Yeah I thought changing that would have done it, but it did not. I changed the flag value anyway just in case it causes confusion in the future.

Reply:
I know the `heading` and `description` are used defo being used